### PR TITLE
NAS-101675 / 11.2 / Generate an alert when creating a new ticket for TrueNAS proactive al…

### PR DESCRIPTION
**Pull request by themylogin**
…ert fails

### CODE REVIEW

File: src/middlewared/middlewared/plugins/alert.py [hunk 1]
```diff
@@ -296,6 +298,18 @@ def restore(self, id):
                                     })
                                 except Exception:
                                     self.logger.error(f"Failed to create a support ticket", exc_info=True)
+                                    await self.middleware.call("mail.send", {
+                                        "cc": ["auto-support@ixsystems.com"],
```
Comment (themylogin) [inline src/middlewared/middlewared/plugins/alert.py]: @william-gr which e-mail we should use here?

File: src/middlewared/middlewared/plugins/alert.py [hunk 2]
```diff
@@ -296,6 +298,18 @@ def restore(self, id):
...
+                                        "subject": "Failed creating automatic alert (%s)" % serial,
```
Comment (Mrt134) [inline src/middlewared/middlewared/plugins/alert.py]:
```suggestion
                                        "subject": "Failed to notify iXsystems about alert (%s)" % serial,
```

File: src/middlewared/middlewared/plugins/alert.py [hunk 3]
```diff
@@ -296,6 +298,18 @@ def restore(self, id):
...
+                                        "text": f"""
+                                            Creating automatic alert for iXsystems for your system {serial} failed
```
Comment (Mrt134) [inline src/middlewared/middlewared/plugins/alert.py]:
```suggestion
                                            Notifying iXsystems about an alert on system {serial} failed.
```

File: src/middlewared/middlewared/plugins/alert.py [hunk 4]
```diff
@@ -296,6 +298,18 @@ def restore(self, id):
...
+                                            for some reason. Please, contact us.
```
Comment (Mrt134) [inline src/middlewared/middlewared/plugins/alert.py]:
```suggestion
                                            Please contact iXsystems Support.
```

File: src/middlewared/middlewared/plugins/alert.py [hunk 5]
```diff
@@ -296,6 +298,18 @@ def restore(self, id):
...
+
+                                            Below is the text of automatic alert:
```
Comment (Mrt134) [inline src/middlewared/middlewared/plugins/alert.py]:
```suggestion
                                            Alert:
```

File: src/middlewared/middlewared/plugins/alert.py (hunk 4 above)
Comment (Mrt134) [reply src/middlewared/middlewared/plugins/alert.py]: Maybe add email for iX support here too?

File: src/middlewared/middlewared/plugins/alert.py (hunk 1 above)
Comment (william-gr) [reply src/middlewared/middlewared/plugins/alert.py]: Waiting on feedback from support people, will get back to you when I have an answer

Comment (yocalebo) [reply src/middlewared/middlewared/plugins/alert.py]: @themylogin please remove the cc line from the `except Exception` block. We will let the customer manually open a ticket in this scenario.

**Pull request merged**